archivr.R

```diff
@@ -158,10 +194,36 @@ view_archiv.fromUrl <- function (url, source="wayback") {
   return(view_archiv(get_urls_from_webpage(url), source))
 }
 
+#' Collect information on whether links in a file are archived.
+#'
+#' @param fp The filepath to extract links from.
+#' @param source Either "wayback," "perma_cc" or "both".
+#' @return a dataframe containing the url, status, availability,
+#'   archived url(s) and timestamp(s)
 view_archiv.fromText <- function (fp, source="wayback") {
   return(view_archiv(extract_urls_from_text(fp), source))
 }
 
+#' Collect information on whether links in a url are archived.
+#'
+#' @param url The url to extract links from.
+#' @param source Either "wayback," "perma_cc" or "both".
+#' @return a dataframe containing the url, status, availability,
+#'   archived url(s) and timestamp(s)
+save_archiv.fromUrl <- function (url, source="wayback") {
+  return(archiv(extract_urls_from_webpage(url), source))
```

Comment:
this needs to be `get_urls_from_webpage` (or, perhaps better, rename the function to `extract...` for uniformity

Reply:
Good idea! Will fix.

Reply:
fixed in this PR